### ISSUE TITLE
[i18n] Quotation marks

### ISSUE THREAD

**Issue by j3ll3yfi5h** [statamic/cms#4164]
## Bug Description

If you write text (e.g. in bard) and you use quotation marks, it always uses the ASCII quotation marks. For other languages like e.g. German, this is not correct.

A solution connected to the locale of the site would be the way to go I think.

## Extra Detail

From https://en.wikipedia.org/wiki/Quotation_mark:
> '…' and "…" are known as neutral, vertical, straight, typewriter, dumb, or ASCII quotation marks. The left and right marks are identical.

## Environment
Statamic 3.1.34 Pro
Laravel 8.55.0
PHP 8.0.8
statamic/collaboration 0.1.0

**Install method (choose one):**
- Fresh install from `statamic/statamic`

## PS
I hate it when clients say: "In WordPress this worked correctly."

**Comment by jasonvarga** [comment]
Are you saying that if you type a quotation mark... it shouldn't use a quotation mark?

Can you show a video of what you're talking about?

**Comment by j3ll3yfi5h** [comment]
Yes, sure! To be clearer: it's the output, which always shows the same quotation marks.

# English
<img width="450" alt="Bildschirmfoto 2021-08-23 um 16 26 37" src="https://user-images.githubusercontent.com/6337411/130464983-92f9e6df-6f0b-4797-9f97-7ba7a9436322.png">

https://user-images.githubusercontent.com/6337411/130465201-c23c2bcc-cc83-48f3-b868-1f6daa72f881.mov

# German

<img width="449" alt="Bildschirmfoto 2021-08-23 um 16 26 17" src="https://user-images.githubusercontent.com/6337411/130465085-0ed4f097-2c4b-4840-a3dc-6a180a819b8c.png">

https://user-images.githubusercontent.com/6337411/130465100-e17d6d79-2152-4177-8396-fe27ce655d08.mov

**Comment by jasonvarga** [comment]
Gotcha! That's neat.

I'd say it's a feature request, not a bug. Right now you get what you enter, which still makes plenty of sense.

Moving this to the ideas repo.
